doc/ice_in

```diff
@@ -65,18 +68,18 @@
   ustar_min = 0.0005
 /
 &domain_nml
-  block_size_x = 16
-  block_size_y = 15
+  block_size_x = 15
+  block_size_y = 20
```

Comment:
As was discussed in https://github.com/COSIMA/access-om3/issues/214#issuecomment-2316682306 and [cice docs](https://cice-consortium-cice.readthedocs.io/en/main/user_guide/ug_implementation.html#performance:~:text=Often%203%20to%208%20blocks%20per%20processor%20provide%20the%20decompositions%20flexiblity%20to%20create%20reasonable%20load%20balance%20configurations.), It is suggested that `max_blocks` be set in the range of 3-8 blocks per processor. 
```suggestion
  block_size_x = 30
  block_size_y = 40
```

With the updated `block_size_x` and `block_size_y`, `max_blocks` is determined to be 8, which suits the recommendation from the [cice docs](https://cice-consortium-cice.readthedocs.io/en/main/user_guide/ug_implementation.html#performance:~:text=Often%203%20to%208%20blocks%20per%20processor%20provide%20the%20decompositions%20flexiblity%20to%20create%20reasonable%20load%20balance%20configurations.).

Reply:
With the slender distributions block_size_x should equal nx_global/nprocs i.e. 360/24 = 15. Which is why these block sizes are set as they are at the moment

Reply:
You could try that. That would mean there is 1 block per processor, which means you couldn't eliminate any blocks which are entirely land. I am not sure how it could impact inter-process communication ? I suspect the performance will be about the same as the current configuration ?

Reply:
I dont get it. If that’s the case, why does block_size_x equal nx_global / nprocs, but block_size_y is calculated differently?

Reply:
Its just kind of by definition, you can have multiple blocks in Y but not X. Each color is a different processor: 

<img width="275" alt="Screenshot 2024-09-03 at 1 36 33 PM" src="https://github.com/user-attachments/assets/103f2bc4-d850-4f59-84a7-2612af926481">